tests/test_tracker.py

```diff
@@ -1,85 +1,46 @@
-        assert tracker.status == status
-        assert tracker.tracking_details != []
-        if status == "delivered":
-            assert tracker.signed_by == "John Tester"
+    assert isinstance(tracker, easypost.Tracker)
+    assert str.startswith(tracker.id, "trk_")
+    assert tracker.status == "pre_transit"
 
 
 @pytest.mark.vcr()
-def test_tracker_interactions(per_run_unique):
-    # create a pseudo-random tracking code so that we can run multiple instances of this test in parallel
-    tracking_code = "EP{0}".format(per_run_unique)
-
-    # Create a tracker and then retrieve it. We assert on created and retrieved tracker's values.
-    tracker = easypost.Tracker.create(
-        tracking_code=tracking_code,
-        carrier="usps",
-        options=dict(
-            full_test_tracker=True,
-        ),
-    )
-    assert tracker.id  # This is random
-
-    # retrieve tracker by id
-    tracker2 = easypost.Tracker.retrieve(tracker.id)
+def test_tracker_retrieve():
+    tracker = easypost.Tracker.create(tracking_code="EZ1000000001")
 
-    assert tracker2.id == tracker.id  # Should be the same as above
+    retrieved_tracker = easypost.Tracker.retrieve(tracker.id)
 
-    # retrieve all trackers by tracking_code
-    trackers = easypost.Tracker.all(tracking_code=tracking_code)
+    assert isinstance(retrieved_tracker, easypost.Tracker)
+    assert retrieved_tracker.id == tracker.id
 
-    assert len(trackers["trackers"])
-    assert trackers["trackers"][0].id == tracker.id == tracker2.id  # Should be the same as the ids above
 
-    # create another test tracker
-    tracker3 = easypost.Tracker.create(
-        tracking_code=tracking_code,
-        carrier="USPS",
-        options=dict(
-            full_test_tracker=True,
-        ),
-    )
-
-    assert tracker3.id
+@pytest.mark.vcr()
+def test_tracker_all(page_size):
+    trackers = easypost.Tracker.all(page_size=page_size)
 
-    # retrieve all created since 'tracker'
-    trackers2 = easypost.Tracker.all(after_id=tracker.id, tracking_code=tracking_code)
+    trackers_array = trackers["trackers"]
 
-    # assert len(trackers2["trackers"]) == 1  this is ignored because the number can vary depending on the user account
-    assert trackers2["has_more"] is not None
-    assert trackers2["trackers"][0].id == tracker3.id  # Should be the same as the id for tracker3
+    assert len(trackers_array) <= page_size
+    assert trackers["has_more"] is not None
+    assert all(isinstance(tracker, easypost.Tracker) for tracker in trackers_array)
 
 
 @pytest.mark.vcr()
 def test_tracker_create_list():
-    """Tests that we can create a list of trackers in bulk."""
-    success = easypost.Tracker.create_list(
+    response = easypost.Tracker.create_list(
         {
             "0": {"tracking_code": "EZ1000000001"},
-            "1": {"tracking_code": "EZ2000000002"},
-            "2": {"tracking_code": "EZ3000000003"},
-            "3": {"tracking_code": "EZ4000000004"},
```

Comment:
This line was valid and could be left.

Reply:
I was using the test input example from other libs to make it consistent, the ` "3": {"tracking_code": "EZ4000000004"},` is unnecessary, let's keep what I have now.